src/UBAFeeCalculator/UBAFeeUtility.ts

```diff
@@ -51,12 +62,14 @@ export function performLinearIntegration(
  * @returns The upper and lower bounds of the interval
  */
 export function getBounds(cutoffArray: [BigNumber, BigNumber][], index: number): [BigNumber, BigNumber] {
+  const largestBound = BigNumber.from(parseUnits((-MAX_SAFE_JS_INT).toString(), 18)).mul(-1);
```

Comment:
I'm curious about the motivation for instantiating a large negative number and then multiplying it by -1. I tested with both and it seems to produce the same result, and the MAX_SAFE_JS_INT already seems to be 2^32 - 1, so -MAX_SAFE_JS_INT is probably not the same as "MIN_SAFE_JS_INT" (if there were one).

Worth a comment to clarify?

Reply:
Updated here: #259